components/brave_wallet/browser/eth_response_parser.cc

```diff
@@ -219,16 +230,22 @@ bool ParseEthGasPrice(const std::string& json, std::string* result) {
   return ParseSingleStringResult(json, result);
 }
 
-bool ParseEnsResolverContentHash(const std::string& json,
-                                 std::string* content_hash) {
-  DCHECK(content_hash);
+bool ParseERC721TokenUri(const std::string encoded, GURL* url) {
+  size_t offset = 2 /* len of "0x" */ + 64 /* len of offset to array */;
+  std::string decoded;
+  brave_wallet::DecodeString(offset, encoded, &decoded);
```

Comment:
I thought we already decode the string in ParseStringResult step?

Reply:
I didn't end up using the new ParseStringResult function for ERC721 metadata.  Some details here https://github.com/brave/brave-core/pull/12594/files/984fa57fe19a535140b3649f2be67fc3749586c7#r835520980.